test/test_sample.py

```diff
@@ -245,6 +245,80 @@ def test_fixed_param_good(self):
         self.assertEqual(datagen_fit.metric_type, None)
         self.assertEqual(datagen_fit.stepsize, None)
 
+    def test_bernoulli_file_with_space(self):
+        stan = os.path.join(datafiles_path, 'bernoulli with space in name.stan')
```

Comment:
Is there a difference with filename with space and filepath with space?

Reply:
Interestingly, there might be. I get different error codes without the patch for the case of a file with spaces vs a directory with spaces. In both cases this PR fixes it. So, I can add another test for a path with spaces.